firefox-ios/firefox-ios-tests/Tests/StorageTests/TestBrowserDB.swift

```diff
@@ -79,8 +87,12 @@ class TestBrowserDB: XCTestCase {
 
         // Grab a pointer to the -shm so we can compare later.
         let shmAAttributes = try files.attributesForFileAt(relativePath: "foo.db-shm")
-        let creationA = shmAAttributes[FileAttributeKey.creationDate] as! Date
-        let inodeA = (shmAAttributes[FileAttributeKey.systemFileNumber] as! NSNumber).uintValue
+        guard let creationA = shmAAttributes[FileAttributeKey.creationDate] as? Date else {
+            return XCTFail("expected object was not a date")
```

Comment:
Here I'd put the `return` statement after `XCTFail` and please start the message with capital letter to be consistent like in the other cases.